api/src/opentrons/simulate.py

```diff
@@ -302,6 +304,7 @@ def simulate(
                               :py:attr:`.ProtocolContext.bundled_data`.
     :param hardware_simulator_file_path: A path to a JSON file defining a
                                          hardware simulator.
+    :param duration_estimator: Optional duration estimator object.
```

Comment:
```suggestion
    :param duration_estimator: For internal use only.
                               Optional duration estimator object.
```

This will unfortunately appear on docs.opentrons.com, so we need to be careful about putting internal technical stuff in front of API users.

(This is a pervasive problem for everything in this module, and it needs more dedicated attention to properly solve. For now, I think we should just avoid adding to the pile, when possible.)